<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSetting</class>
 <widget class="QDialog" name="dlgSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rms_update.qrc">
    <normaloff>:/images/setting.png</normaloff>:/images/setting.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="rms_update.qrc">:/images/regional.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="LanguageComboBox">
       <item>
        <property name="text">
         <string>Simplified Chinese</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>English American</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="rms_update.qrc">
         <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="rms_update.qrc">
         <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rms_update.qrc"/>
 </resources>
 <connections/>
</ui>
